tests/test_assets.py

```diff
@@ -181,3 +184,63 @@ def test_timeout(clean_assets):
 
     with pytest.raises(concurrent.futures.TimeoutError):
         a.load(timeout=0.1)
+
+
+def test_chained(clean_assets):
+    class Const(BackgroundMixin, AbstractAsset):
```

Comment:
This is kind of magical, can we get a comment that this is essentially a threaded string.join so folks at least get the intent?